api/pacman-api-asset/src/main/java/com/tmobile/pacman/api/asset/domain/NotificationRequest.java

```diff
@@ -0,0 +1,169 @@
+package com.tmobile.pacman.api.asset.domain;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.base.Joiner;
+public class NotificationRequest {
+
+    private String searchtext = null;
+
+    private int from;
+
+    private int size;
+
+    private Map<String, List<String>> filter;
+
+    private Map<String, Object> reqFilter;
+
+    private Map<String, Object> sortFilter;
+
+    private String ag;
```

Comment:
let's have a better naming for variables. what is this ag? asset group I assume? is its name or id or anything else?

Reply:
done